pkg/oc/cli/cancelbuild/cancelbuild.go

```diff
@@ -137,12 +139,19 @@ func (o *CancelBuildOptions) Complete(f kcmdutil.Factory, cmd *cobra.Command, ar
 	if err != nil {
 		return err
 	}
-	o.Client, err = buildinternalclient.NewForConfig(config)
+
+	o.Client, err = buildtypedclient.NewForConfig(config)
+	if err != nil {
+		return err
+	}
+
+	internalclient, err := buildinternalclient.NewForConfig(config)
 	if err != nil {
 		return err
 	}
-	o.BuildLister = buildclient.NewClientBuildLister(o.Client.Build())
-	o.BuildClient = o.Client.Build().Builds(o.Namespace)
+
+	o.BuildLister = buildclientinternal.NewClientBuildLister(internalclient.Build())
```

Comment:
@soltysh @deads2k I'm using an internal clientBuildLister since we do not have a `buildutil.BuildConfigBuilds` helper that can deal with external versions